src/lubed/cli.py

```diff
@@ -301,15 +302,20 @@ def create_issue(last_timestamp_file, config_path, gh_token, no_update_timestamp
     )
 
     with console.status("Creating issue...", spinner="arc"):
-        url = gh.create_issue_in_board(
+        issue = gh.create_issue_in_board(
             repo_name=gh_repo,
             title=issue_title,
             body=issue_body,
             label_names=issue_labels,
             gh_token=gh_token,
             column_id=gh_column_id,
         )
-    console.print(f"View the issue at {url}")
+
+        issue_node_id = gh.get_issue_node_id(issue.number, gh_repo, gh_token)
+
+        gh.assign_issue_to_board(issue_node_id, gh_project_board_id, gh_token)
```

Comment:
[nitpick]: I would do this from within `create_issue_in_board` since that's how it was done up to now (and it's in the name)